PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,25 @@
-<!-- /\ Please prefix the subject above with the Jira issue number. Example: -->
-<!-- [PROJECT-NNN] Fixes an issue where ... -->
-<!-- This should connect your PR to the appropriate JIRA ticket -->
+## Description
+<!-- Please include a summary of the change, motivation, and context. Screenshots are encouraged. -->
 
-<!-- Please update the following with your Jira project and issue number. -->
-# [PROJECT-NNN] Ticket Title / Description
```

Comment:
Thank you, this was duplicate info

Reply:
Is it? Does the jira bot link to the issue when it's in the PR title?

Reply:
Confirmed yes

![image](https://user-images.githubusercontent.com/824632/76356238-9c717180-62e3-11ea-8e7d-300595479046.png)